README.md

```diff
@@ -20,4 +25,28 @@ and then run the server locally with:
 bun run dev --host
 ```
 
-Then you should see a prompt with the local IP and port that can be accessed through a browser, such as `localhost:5173`.
\ No newline at end of file
+Then you should see a prompt with the local IP and port that can be accessed through a browser, such as `localhost:5173`.
+
+## Installing backend providers
+
+Cockpit is currently a frontend-only application. That means it relies on data offered by other backend solutions, which are not built-in.
+
+If you have a [BlueOS](https://github.com/bluerobotics/BlueOS) instance running, there's no need to install any backend, as BlueOS already provides everything.
+
+If that's not your case and you want to install the necessary backends, follow the instructions below:
+
+1. Install a MAVLink router. We recommend [mavp2p](https://github.com/bluenviron/mavp2p) or [MAVLink Router](https://github.com/mavlink-router/mavlink-router).
+2. Run the MAVLink router, connecting it to your vehicle or [SITL instance](https://ardupilot.org/dev/docs/sitl-simulator-software-in-the-loop.html).
+3. Install [mavlink2rest](https://github.com/mavlink/mavlink2rest).
+4. Run `mavlink2rest`, pointing it to the endpoint provided by the MAVLink router. Remember to serve the API over the 6040 port by running it with `--server 127.0.0.1:6040`.
```

Comment:
I dont mind servers at 0.0.0.0, as that means "bind to all interfaces available". Just connecting to 0.0.0.0 as a client that is conceptually... dubious

Reply:
I changed that one by mistake 🤦 
That's exactly the case for 0.0.0.0 haha
Fixing before merging.
Good catch.